scripts/tests/test_extract_resume_linkedin.py

```
@@ -0,0 +1,477 @@
+        joined = "\n".join(result)
+        self.assertIn("Acme Corp", joined)
+        self.assertIn("Startup Inc", joined)
+
+    def test_each_role_has_position_as_section_header(self):
+        result = build_experience(self.data)
+        self.assertIn("## Senior Developer", result)
+        self.assertIn("## Developer", result)
+
+    def test_includes_company_and_dates(self):
+        result = build_experience(self.data)
+        self.assertIn("Acme Corp, Jan 2020 – Jun 2023", result)
```

Ruff (0.15.20)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)